modules/ROOT/pages/download-and-install-service-mesh.adoc

```diff
@@ -46,4 +46,42 @@ Anypoint Service Mesh is installed and a status of all pods is displayed:
 +
 image::verified-service-mesh-installer-output.png[]
 
+=== Install With Proxy
+
+You might need to install Anypoint Service Mesh in a proxy environment. In such cases, you must provide provide the proxy configuration during the installation.
+
+To install Anypoint Service Mesh with proxy configuration:
+
+. From the location where you downloaded your Service Mesh client, type:
++
+`./service-mesh install --clientId=<clientId> --clientSecret=<clientSecret> --license=<license absolute or relative path> --platformUri=<Anypoint Platform Uri> --proxy.host=<host> --proxy.protocol=<protocol> --proxy.port=<port> --proxy.user=<proxy username> --proxy.password=<proxy password>`
++
+[%header%autowidth.spread]
+|===
+| Parameter Name | Description | Required/Optional | Default Value
+| `clientId` | The client ID of the environment (recommended) or the organization. Use the client ID that you obtained as part of the permission and roles prerequisites as explained in this document. | Required | N/A
+| `clientSecret` | The client secret of the environment (recommended) or the organization. Use the client secret that you obtained as part of the permission and roles prerequisites as explained in this document.| Required | N/A
+| `license` | The Mule license you obtained as part of the prerequisites. Provide the path to the location where you license is stored. | Required | N/A
+| `platformUri` | The URL of the proxy.| Optional | https://anypoint.mulesoft.com
+| `proxy.host` | The name of the proxy host. | Optional | default
+| `proxy.protocol`| The protocol used by the proxy, for example, HTTP or WSDL. | Optional | default
```

Comment:
It does not make sense that the value for the Dafault Value column is "default" in these two table rows.

Reply:
Yes, confirmed! Thanks!